source/chapters/user_interface.rst

```diff
@@ -65,7 +68,7 @@ The Deck Sections
    RJ Ryan <rryan@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
 
-.. figure:: ../_static/2.2/interface-decks.png
+.. figure:: ../_static/2.3/deck-sections.png
```

Comment:
I will explain this in full length in the LateNight blogpost, but it should at least be mentioned here because when users hide the mixer the decks will by default switch to medium decks to make the feature somewhat discoverable.